src/js/subtraction/components/Attribution.tsx

```diff
@@ -3,6 +3,11 @@ import styled from "styled-components";
 import { getColor, getFontSize, theme } from "../../app/theme";
 import { Attribution, AttributionWithName } from "../../base";
 
+/**
+ * Styled component for the attribution when none found.
+ * @prop color - The color of the text.
+ * @prop theme - The theme object.
```

Comment:
@props is tot part of the tsdoc standard. https://tsdoc.org/. Create a type and document the props there

Reply:
My mistake - they were actually not props